tests/unit/output-parser.test.ts

```diff
@@ -0,0 +1,498 @@
+                        "ports": [
+                          {
+                            "containerPort": 8080,
+                            "name": "http1"
+                          }
+                        ],
+                        "resources": {
+                          "limits": {
+                            "cpu": "1000m",
+                            "memory": "512Mi"
+                          }
+                        }
+                      }
+                    ],
+                    "serviceAccountName": "392520182231-compute@developer.gserviceaccount.com",
+                    "timeoutSeconds": 300
+                  }
+                },
+                "traffic": [
+                  {
+                    "percent": 100,
+                    "revisionName": "hello-00007-jec"
+                  },
+                  {
+                    "revisionName": "hello-00005-zay",
+                    "tag": "test"
+                  },
+                  {
+                    "revisionName": "hello-00007-jec",
+                    "tag": "another"
+                  }
+                ]
+              },
+              "status": {
+                "address": {
+                  "url": "https://hello-4goqgbaxqq-uc.a.run.app"
+                },
+                "conditions": [
+                  {
+                    "lastTransitionTime": "2022-01-25T22:43:07.210548Z",
+                    "status": "True",
+                    "type": "Ready"
+                  },
+                  {
+                    "lastTransitionTime": "2022-01-25T22:43:07.210548Z",
+                    "status": "True",
+                    "type": "ConfigurationsReady"
+                  },
+                  {
+                    "lastTransitionTime": "2022-01-25T22:41:16.384919Z",
+                    "status": "True",
+                    "type": "RoutesReady"
+                  }
+                ],
+                "latestCreatedRevisionName": "hello-suffix",
+                "latestReadyRevisionName": "hello-suffix",
+                "observedGeneration": 9,
+                "traffic": [
+                  {
+                    "percent": 100,
+                    "revisionName": "hello-00007-jec"
+                  },
+                  {
+                    "revisionName": "hello-00005-zay",
+                    "tag": "test",
+                    "url": "https://test---hello-4goqgbaxqq-uc.a.run.app"
+                  },
+                  {
+                    "revisionName": "hello-00007-jec",
+                    "tag": "another",
+                    "url": "https://another---hello-4goqgbaxqq-uc.a.run.app"
+                  }
+                ],
+                "url": "https://hello-4goqgbaxqq-uc.a.run.app"
+              }
+            }
+          `,
+        },
+        expected: 'https://test---hello-4goqgbaxqq-uc.a.run.app',
+      },
+      {
+        name: 'handles invalid stdout',
```

Comment:
Can we add a test for the response being `[]` and for the response being the empty string?